GliaWidgets/Component/AttachmentList/AttachmentSourceListStyle.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+/// Style of the list that contains the attachment sources. Appears in media upload menu popover.
+public class AttachmentSourceListStyle {
+    /// Items to show in the list.
+    public var items: [AttachmentSourceItemStyle]
+
+    /// Color of the seprator line.
+    public var separatorColor: UIColor
+
+    /// Background color of the view.
```

Comment:
There are indeed default colors - the ones provided in the design. We can obviously put them there, but in what format? RGBA? hex?
If _any_ property is undefined, it uses the default one, not just the colors, but fonts, texts etc. as well. I think it would be too much to mention it on every property, but we can put it somewhere in the integration guide.